src/main/scala/Main.scala

```diff
@@ -2,7 +2,30 @@
   * Created by Administrator on 05/06/2017.
   */
 object Main extends App {
-
+  def createString(stringed: String, style: String, loop: Int): String = {
+    if(stringed.charAt(0) != '-'){
```

Comment:
nesting if elses is a bad practice, this could be reworked into pattern matching